invokeai/app/invocations/param_easing.py

```diff
@@ -0,0 +1,232 @@
+import io
+from typing import Literal, Optional
+
+# from PIL.Image import Image
+import PIL.Image
+from matplotlib.ticker import MaxNLocator
```

Comment:
I don't think we have matplotlib as an install requirement. But we do have it installed as a dependency of mediapipe or one of the other libs. If we ever stop using them, this will break. So maybe add matplotlib as a dependency.. ?

Reply:
Added